tests/bluetooth/shell/boards/native_posix.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_LOG_MODE_IMMEDIATE=y
+CONFIG_NO_OPTIMIZATIONS=y
+# Allows for copying larger amount of data into the shell
+CONFIG_SHELL_BACKEND_SERIAL_RX_RING_BUFFER_SIZE=4096
+#Enable TINYCRYPT for native_posix; needed for CSIS
+CONFIG_BT_TINYCRYPT_ECC=y
+
+# For native posix k_sleep is used in the data path as well as for shell input
+# detection, hence data processing is at least two ticks per packet. To support
+# 5ms ISO interval bidirectional data the system shall never stall for more
+# than 1.5 ms in average.
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=500
```

Comment:
Missing newline at end of file

Reply:
@Casper-Bonde-Bose Still looks like it's missing the empty newline.

Reply:
Yes, I have changed it locally, but are awaiting OK from Luiz on the API changes before I upload again - to avoid too many notifications to be send to people..